bin/handler-mailer.rb

```diff
@@ -34,8 +34,8 @@ def self.call(path, arguments, _destinations, encoded_message)
 end
 
 class Mailer < Sensu::Handler
-  option :json_config,
-         description: 'Config Name',
+  option :json_config_name,
```

Comment:
Unless I'm missing something changing the name of the option wouldn't be breaking as nobody uses that directly. The README has an example of the json configuration. Perhaps we should add a note there that you can set the key of the config using the `-j` flag. I do like the description change on the option that makes it more clear.